lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/repeat-fraud-check.yaml

```diff
@@ -0,0 +1,138 @@
+# Entry points
+
+START:
+  events:
+    next:
+      targetState: CONFIRM_NAME_DOB
+
+# Parent States
+
+CRI_STATE:
+  events:
+    not-found:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    fail-with-no-ci:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    error:
+      targetJourney: TECHNICAL_ERROR
+      targetState: ERROR
+    access-denied:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    enhanced-verification:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    temporarily-unavailable:
+      targetJourney: TECHNICAL_ERROR
+      targetState: ERROR
+    fail-with-ci:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    vcs-not-correlated:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+    alternate-doc-invalid-dl:
+      targetJourney: FAILED
+      targetState: FAILED_RFC
+
+# Journey States
+
+CONFIRM_NAME_DOB:
+  response:
+    type: page
+    pageId: pyi-confirm-name-dob
+  events:
+    next:
+      targetState: CONFIRM_ADDRESS
+    end:
+      targetState: UPDATE_NAME_DOB
+
+CONFIRM_ADDRESS:
+  response:
+    type: page
+    pageId: pyi-confirm-address
+  events:
+    addressCurrent:
```

Comment:
Minor but our other events are all kebab-case (e.g. `access-denied`), I appreciate that you've changed it though!